token/cli/src/main.rs

```diff
@@ -171,6 +171,13 @@ pub fn multisig_signer_arg<'a, 'b>() -> Arg<'a, 'b> {
         .help(MULTISIG_SIGNER_ARG.help)
 }
 
+pub fn memo_arg<'a, 'b>() -> Arg<'a, 'b> {
+    Arg::with_name("memo")
+        .long("with-memo")
+        .takes_value(true)
+        .help("Specify a memo string to include in the transaction.")
+}
```

Comment:
Import this from solana-clap-utils::memo?

Reply:
ooo, thanks